tests/cassettes/TestRoutes.test_homepage.yaml

```diff
@@ -1,1908 +0,0 @@
-interactions:
```

Comment:
How come you're deleting this cassette? The test [still exists](https://github.com/carkod/ubuntu.com/blob/cleanup-authentication/tests/test_routes.py#L28) I think...

Reply:
Well, I don't know why, but I added it because with authentication the test was failing. Now we can test `/` without authentication, do we still want to do it against mocks?

Reply:
I just put it back. Merging